packages/esm-patient-forms-app/src/htmlformentry/html-form-entry-wrapper.component.tsx

```diff
@@ -18,17 +20,26 @@ const HtmlFormEntryWrapper: React.FC<HtmlFormEntryWrapperProps> = ({ closeWorksp
 
   // hide the headers and breadcrumbs of the O2 page on load
   const onLoad = useCallback(() => {
+    setIsIframeLoading(false);
+    iframeRef.current.contentWindow.addEventListener('beforeunload', () => setIsIframeLoading(true));
     const iframe = iframeRef.current;
     const dashboard = iframeRef.current.contentDocument;
     dashboard.querySelector('header')?.remove();
     dashboard.querySelector('.patient-header')?.remove();
     dashboard.querySelector('#breadcrumbs')?.remove();
     iframe.style.display = 'block';
     iframe.height = dashboard.body.scrollHeight + 50 + 'px'; // set iframe height to the height of the form, with some padding
+
+    const styleTag = dashboard.createElement('style');
+    styleTag.innerHTML = `@media screen {
+      body { background: white; }
+     }`;
+    dashboard.head.appendChild(styleTag);
```

Comment:
I also added this chunk of css in to remove the stray grey background at the top of the htmlform when it is loaded into the workspace.